linux/dump_service_no_lcd.service

```diff
@@ -4,7 +4,7 @@ After=mnt-source.mount
 
 [Service]
 Type=idle
-ExecStart=/usr/bin/sudo /usr/bin/python3 /home/pi/dump/linux/dumper/dumper_no_lcd.py
+ExecStart=/usr/bin/sudo /usr/bin/python3 /home/pi/dump/linux/dumper/dumper.py monitor
```

Comment:
We are going to remove this functionality. We don't have to make any changes, in fact we should probably just remove these *no_lcd.* files.